packages/ecommerce-app-base/src/interfaces.ts

```diff
@@ -45,6 +45,7 @@ export interface ParameterDefinition {
 
 export interface Product {
   sku: string;
+  displaySKU?: string;
```

Comment:
Can this ever be `undefined`? I think it's always either `SKU: ...` or `Product ID: ...`.
If it's never `undefined`, you could also remove some  `|| ${product.sku}` from the components.

Reply:
I'm thinking of the other apps that use this interface at the moment, and possible external users, as it would be a breaking change to make it required - and I don't know if it is necessary.